external/jtreg-buffer/test.properties

```diff
@@ -0,0 +1,5 @@
+github_url="https://github.com/rh-openjdk/jtreg-buffer.git"
+test_results=""
+tag_version="main"
+ubuntu_packages="git wget tar"
+maven_version="3.8.5"
```

Comment:
would prefer 3.9.6 if possible.

Reply:
Actually this do not need maven at all. Will drop that line. If it is mandatory for sute to be picked up, I willreturn it with 3.9.6